docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,125 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 
 # Prerequisites for .NET Core on Windows
 
-This article shows you what dependencies you need to deploy and run .NET Core applications on Windows machines and develop using Visual Studio.
+This article shows the needed dependencies to develop .NET Core applications on Windows. The supported OS versions and dependencies that follow apply to the three ways of developing .NET Core apps on Windows:
 
-## Supported Windows versions
+* [Command line](tutorials/using-with-xplat-cli.md)
 
-.NET Core is supported on the following versions of Windows:
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/)
+
+* [Visual Studio Code](https://code.visualstudio.com/)
+
+# [.NET Core 1.x](#tab/netcore1x)
```

Comment:
Ok, I think that's a typo on the page. I'll submit a PR. AFAIK, only 2016 has a nano version.

https://docs.microsoft.com/en-us/windows-server/get-started/getting-started-with-nano-server